### PR TITLE
Feat/support feedback

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/components/Forms/PaymentForm/Processing/index.tsx [hunk 1]
```diff
@@ -196,24 +201,22 @@ const USDTProcessingForm: React.FC<Props> = ({
...
-        currency,
-        purpose: 'donation',
-        recipientId: recipient.id,
-        targetId,
-        chain: CHAIN.POLYGON,
-        txHash: data.hash,
-      },
-    })
-
-    await data.wait()
```
Comment (robertu7) [inline src/components/Forms/PaymentForm/Processing/index.tsx:216]: any reason to remove the tx confirmation?

File: src/views/ArticleDetail/SupportWidget/Animation.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
+import React, { useEffect } from 'react'
+import { useInView } from 'react-intersection-observer'
```
Comment (robertu7) [inline src/views/ArticleDetail/SupportWidget/Animation.tsx:2]: I think we can resue `react-waypoint` instead?

https://github.com/thematters/matters-web/blob/develop/package.json#L107

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/views/ArticleDetail/SupportWidget/Animation.tsx (hunk 2 above)
Comment (pitb2022) [reply src/views/ArticleDetail/SupportWidget/Animation.tsx]: 👌

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/views/ArticleDetail/SupportWidget/index.tsx [hunk 3]
```diff
@@ -1,55 +1,267 @@
...
+        },
+      })
+      await transactionResult.wait()
+      setPlayShipWaiting(false)
+      hasDonatedRefetch()
+    }
+  )
+
   return (
     <section className={supportWidgetClasses}>
```
Comment (robertu7) [inline src/views/ArticleDetail/SupportWidget/index.tsx:132]: border radius is not correct as design.

image

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/views/ArticleDetail/SupportWidget/Donators/index.tsx [hunk 4]
```diff
@@ -24,32 +31,94 @@ const Donators = ({ article }: DonatorsProps) => {
...
+            <section className="avatar-list">
+              {donators.map((user, index) => {
+                return (
+                  <>
+                    {showAvatarAnimation && (
+                      <>
+                        {index === 0 && (
+                          <animated.div
+                            style={{
+                              width: '1.25rem',
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/views/ArticleDetail/SupportWidget/Donators/index.tsx:60]: <animated.Avatar> need these styles to align other avatar

**Review by robertu7 — APPROVED** [review]

**Pull request merged**